.rspec

```diff
@@ -1,2 +1,3 @@
 --color
 --require spec_helper
+--format documentation
```

Comment:
whole group agreed to change it to documentation (you can drop those file from git repo) stop tracking it if every member of team wanna to have own style (config)